src/views/product-landing/components/get-started/index.tsx

```diff
@@ -18,7 +20,13 @@ function GetStarted({
     <div className={s.root}>
       <div className={s.container}>
         <div className={s.iconSection}>
-          <InlineSvg src={iconSvg} />
+          {product == 'sentinel' || product == 'hcp' ? (
+            <IconTile size="extra-large" brandColor="neutral-dark">
+              <IconHashicorp16 />
```

Comment:
This conditional for `sentinel` and `hcp` (as opposed to using `IconTileLogo`) felt necessary in order to allow the `HashiCorp` icon to be colored by `IconTile`'s `brandColor` prop.

Reply:
Yep - the intent here is to implement the placeholder content design has come up with, on [the `Key screens: Content discussion II` page in the `Dev Portal` Figma file](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=4486%3A77031):

<img width="1308" alt="dark-product-tile-hcp" src="https://user-images.githubusercontent.com/4624598/153254535-c5c559a1-705a-4d96-9912-89d77c518803.png">

<img width="1311" alt="dark-product-tile-sentinel" src="https://user-images.githubusercontent.com/4624598/153254562-512704f7-3a40-46f1-b6db-044546e6509e.png">



These designs don't specify the name `neutral-dark`, but I figured it might make sense as a name here. Happy to adjust - eg we could name this `temp-neutral-dark` or something to signal that it's been implemented with content review in mind (rather than long-term use).